backend/dataall/cdkproxy/stacks/policies/aws_cicd.py

```diff
@@ -2,26 +2,52 @@
 from aws_cdk import aws_iam as iam
 
 
-class CodeStar(ServicePolicy):
+class AwsCICD(ServicePolicy):
+    """
+    Class including all permissions needed to work with AWS CICD services: CodeCommit, CodePipeline and CodeBuild.
+    It allows data.all users to:
+    - Create and manage CodeBuild, CodeCommit and CodePipeline resources for the team
+    - Create an S3 Bucket for codepipeline prefixed by "codepipeline-"
+    - Read/Write to and from S3 Buckets prefixed by "codepipeline-"
+    """
     def get_statements(self):
         statements = [
             iam.PolicyStatement(
+                #sid="GenericCodeCommit",
                 actions=[
-                    'codecommit:ListRepositoriesForApprovalRuleTemplate',
+                    'codecommit:List*',
                     'codecommit:CreateApprovalRuleTemplate',
                     'codecommit:UpdateApprovalRuleTemplateName',
                     'codecommit:GetApprovalRuleTemplate',
-                    'codecommit:ListApprovalRuleTemplates',
                     'codecommit:DeleteApprovalRuleTemplate',
-                    'codecommit:ListRepositories',
                     'codecommit:UpdateApprovalRuleTemplateContent',
                     'codecommit:UpdateApprovalRuleTemplateDescription',
                 ],
                 resources=['*'],
```

Comment:
Shouldn't it be a dataall/resource specific? 
A malicious user can use data.all to compromise all repos in the account.

Reply:
Some actions (usually List actions) cannot be filtered by resource

Reply:
That might be minor, but we could extract all update/delete/add operations like `UpdateApprovalRuleTemplateName`, `UpdateApprovalRuleTemplateContent` and etc from `*`